src/index.js

```diff
@@ -1,26 +1,79 @@
 import './style.css';
 
-const tasks = [
-  {
-    description: 'My project',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'wash dish',
-    completed: true,
-    index: 2,
-  },
-  {
-    description: 'go to gym',
-    completed: true,
-    index: 3,
-  },
-];
+class TaskObj {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
 
 const myTasksList = document.querySelector('.tasks-list');
+/* eslint-disable max-classes-per-file: "error" */
+class AddRemove {
+  tasks = [];
 
-tasks.forEach((element) => {
-  myTasksList.innerHTML += `<li><input type="checkbox" name="task" id="my-task">
-                             <label for="task">${element.description}</label></li>`;
+  addTask(myTask) {
+    this.tasks.push(myTask);
+    this.#updateList(myTask);
+  }
+
+  removeTask(myTask) {
+    this.tasks = this.tasks.filter((oneTask) => oneTask.description !== myTask.description);
+
+    this.#updateList();
+  }
+
+  #updateList() {
+    myTasksList.innerHTML = '';
+
+    let TaskHtml = '';
+
+    for (let i = 0; i < this.tasks.length; i += 1) {
+      const taskItem = myTasksList.appendChild(document.createElement('li'));
+      taskItem.classList.add('a-task');
+      TaskHtml = `<input type="checkbox" id = "${this.tasks[i].completed} ${this.tasks[i].index}" name="task" id="my-task">
+      <input type="text" id = ${i} class="li-description" value = "${this.tasks[i].description}" required> <i id = ${i} class="fa fa-trash remove-btn" aria-hidden="true"></i>`;
+      taskItem.innerHTML = TaskHtml;
+
+      this.tasks[i].index = i + 1;
+      this.tasks[i].completed = false;
```

Comment:
- [ ] The **checkbox** should have a completed status is checked else it's false, and also if the task is checked it can be removed via the button clear all completed.

Reply:
Hello @dasileker  

Thank you for your feedback, 
you are doing a great job, I really appreciate it but the changes that you request are for the next part of this project. For this one there is no any functionality for the clear all button and the completed value shoud be set by default at false as required. Kindly try to read all the requirements of this part of project and you should notice that all works well as expected 

Thank you !!